### PR TITLE
Add load script

### PR DESCRIPTION
**Pull request by BenBeard115**
Load script added with duplicate database entries removed and general polishing complete.

### CODE REVIEW

File: Pipeline/load.py [hunk 1]
```diff
@@ -0,0 +1,240 @@
+"""Small example of connecting to an MSSQL database."""
```
Comment (IshikaM19) [inline Pipeline/load.py:1]: Can probably change this docstring to something a bit more specific e.g ""Script that loads to the MSSQL database and uploads....."

File: Pipeline/load.py [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+
+    # Remove duplicates
+    insert_data = list(set([i for i in location_tuples]))
+
+    for location in insert_data:
+        for query_location in location_query_data:
+            if float(location[0]) == float(query_location[0]) and float(
+                    location[1]) == float(query_location[1]):
+                insert_data.pop(insert_data.index(location))
+                break
```
Comment (IshikaM19) [inline Pipeline/load.py:108]: What are these breaks for?

Comment (BenBeard115) [reply Pipeline/load.py:108]: For now, it is incase duplicates are already in the database whilst the pipeline is being made. When complete, it'll be just for speed.

File: Pipeline/load.py [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+import pyodbc
+
+
+from extract import extract_main
+from transform import transform_main
+
+
+def get_connection() -> pyodbc.Connection | None:
+    """Connects to the database"""
+    conn_str = f'DRIVER={{ODBC Driver 17 for SQL Server}};SERVER={environ["DB_HOST"]};DATABASE={environ["DB_NAME"]};UID={environ["DB_USER"]};PWD={environ["DB_PASSWORD"]}'
```
Comment (IshikaM19) [inline Pipeline/load.py:14]: Could have this as a variable under the main() function, once you've loaded the load_dotenv(), then you can pass it into this function as an argument? But if you prefer this way that's also fine :)

**Review by IshikaM19 — APPROVED** [review]

**Pull request merged**